packages/core/src/extensions/FormattingToolbar/FormattingToolbarPlugin.ts

```diff
@@ -53,27 +54,29 @@ export class FormattingToolbarView {
 
   public prevWasEditable: boolean | null = null;
 
-  public shouldShow: Exclude<FormattingToolbarPluginProps["shouldShow"], null> =
-    ({ view, state, from, to }) => {
-      const { doc, selection } = state;
-      const { empty } = selection;
+  public shouldShow: Exclude<
+    FormattingToolbarPluginProps<BSchema>["shouldShow"],
```

Comment:
Yeah it's very odd, surprised we didn't see this earlier. `shouldShow` can either be a function or `null`, so it just makes sure its a function. But since, `shouldShow` is also optional, I think it's unnecessary